base/array.jl

```diff
@@ -2412,6 +2412,68 @@ findall(x::Bool) = x ? [1] : Vector{Int}()
 findall(testf::Function, x::Number) = testf(x) ? [1] : Vector{Int}()
 findall(p::Fix2{typeof(in)}, x::Number) = x in p.x ? [1] : Vector{Int}()
 
+"""
+    findsubseq(A,B)
+
+Return the indices of one or many existing subsequences B in A.
+A and B may be of type AbstractString or AbstractArray.
+If there are none in A, return an empty array.
+
+See also: [`issubseq`](@ref).
+
+# Examples
+```jldoctest
+julia> findsubseq([7, 2, 7, 8, 5, 9, 7, 8],[7, 8])
+2-element Vector{Any}:
+ 3
+ 7
+
+julia> findsubseq([7, 2, 7, 8, 5, 9, 7, 8],[4, 6])
+Any[]
+
+julia> findsubseq("Hello","el")
+1-element Vector{Any}:
+ 2
+```
+
+"""
+function findsubseq(A::AbstractArray, B::AbstractArray)
+    BinA = findall(isequal(B[1]), A)
+    matchFirstIndex = []
```

Comment:
This should be concretely typed, something like `eltype(eachindex(A))[]`

Reply:
I agree, i added your suggestion in the code